Lib/sqlite3/deprecated.py

```diff
@@ -0,0 +1,10 @@
+def __getattr__(name):
```

Comment:
Can't you put the code directly in sqlite3/__init__.py?

Reply:
Sure. I used a separate file to make it more explicit.